aspnetcore/release-notes/aspnetcore-9/includes/current_component.md

```diff
@@ -0,0 +1,17 @@
+## Detect the current component's render mode at runtime
```

Comment:
@Rick-Anderson ... You can merge this part as-is without review if you want. I'll have a separate PR this evening for Blazor. I'll polish it up then.